src/decoders/arw.rs

```diff
@@ -75,6 +67,7 @@ impl<'a> Decoder for ArwDecoder<'a> {
       },
       _ => return Err(format!("ARW: Don't know how to decode type {}", compression).to_string()),
     };
+    let orientation = Orientation::from_tiff_ifd(&self.tiff);
```

Comment:
We just need to clone the camera instead of returning the one in the HashMap. The one in the hash is the one with the defaults, the clone is the one that we alter with the metadata from this camera.